tests/foreman/cli/test_remoteexecution.py

```diff
@@ -168,7 +173,25 @@ def test_positive_view_dump(self):
 class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
-    @stubbed
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization which can be re-used in tests."""
+        super(RemoteExecutionTestCase, cls).setUpClass()
+        cls.organization = make_org()
+        ssh.command(
+            '''echo 'echo "test"' > {0}'''.format(TEMPLATE_FILE)
```

Comment:
Ah I see, I realized that would be the same as using the predefined template in *test_positive_run_default_job_template*, so I wanted something different for the custom one. I see it's not very sophisticated, I'll look around for someth. better